Makefile

```diff
@@ -62,6 +67,7 @@ BUILD_TIMESTAMP := $(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
 BUILD_VERSION := $(shell sed -nE "s/^version = \"(.+)\"\$$/\1/p" bin/Cargo.toml)
 BUILD_TAG ?= $(VCS_REF)
 IMAGE_TAG := $(BUILD_TAG)-$(ARCH)
+BUILD_NUMBER := 0
```

Comment:
Is it intended that BUILD_NUMBER is always 0? Or do we want to use `?=` here?

Reply:
thanks!